### PR TITLE
Add example of searching user by name

### PR DESCRIPTION
**Pull request by earthquakesan**
Provide an end-to-end example of how to initialize users request with query parameters and search query. Reproduce the same behavior as in the Azure AD web console search.

### CODE REVIEW

File: docs/users_samples.md [hunk 1]
```diff
@@ -49,4 +49,50 @@ async def get_memberships():
...
+
+```py
+import asyncio
+
+from azure.identity import AzureCliCredential
+from msgraph import GraphServiceClient
+from msgraph.generated.users.users_request_builder import UsersRequestBuilder
+
+
+async def get_user(user_name: str, client: GraphServiceClient) -> None:
```
Comment (baywet) [inline docs/users_samples.md]: could you rename the method to find user instead since this is not getting but rather finding the user?

**Review by baywet — CHANGES_REQUESTED** [review]
Thanks for the great contribution!
@samwelkanda for additional review

File: docs/users_samples.md (hunk 1 above)
Comment (earthquakesan) [reply docs/users_samples.md]: @baywet done! lemme know if there is anything else I can improve in the snippet

**Review by baywet — APPROVED** [review]
Thanks for the contribution. @samwelkanda for final review and merge

**Review by samwelkanda — APPROVED** [review]

**Pull request merged**